javascript/javascript1/week1/homework1JS.js

```diff
@@ -0,0 +1,50 @@
+/*Age-ify 
+(A future age calculator)*/
+
+var yearOfBirth = 1995;
+var yearFuture = 2033;
+var age = yearFuture - yearOfBirth ;
+
+console.log ("You will be " + age + " in" + yearFuture + ".");
+
+
+/*Goodboy-Oldboy 
+(A dog age calculator)*/
+
+
+var dogYearOfBirth = 2018;
+var dogYearFuture = 2023;
+var humanAge = dogYearFuture - dogYearOfBirth;
```

Comment:
```suggestion
var humanAge = humanYearFuture - humanYearOfBirth;
```

I think it could b better to change the variable names to reflect better that you compute human age and not dog age.